libs/coin-modules/coin-tezos/src/logic/listOperations.ts

```diff
@@ -48,7 +58,12 @@ function convertOperation(
     value: BigInt(amount),
     // storageFee for transaction is always present
     fee: BigInt(storageFee ?? 0),
-    blockHeight: 0, // operation.block is a string
+    blockHeight: block.level,
+    block: {
+      hash: block.hash,
```

Comment:
By looking at [tztk doc](https://api.tzkt.io/#operation/Accounts_GetOperations), it seems you can have those `block` info, directly from the account request.
You may don't need to add extra calls. The Datamodel on our side is maybe incorrect.